src/io.jl

```diff
@@ -71,6 +71,36 @@ function close(file::FileHandle)
     return nothing
 end
 
+# Info
+"""
+    MPI.File.get_info(file::FileHandle)
+
+Returns the hints for a file that are actually being used by MPI.
+
+# External links
+$(_doc_external("MPI_File_get_info"))
+"""
+function get_info(file::FileHandle)
+    file_info = Info(init=true)
```

Comment:
Should the `Info` object be initialized? The spec suggests that the returner creates a new object:
https://www.mpi-forum.org/docs/mpi-3.1/mpi31-report/node315.htm#Node315

Reply:
Good catch.  It wasn't clear if the caller or callee allocated the object from the mpich api docs.